term_posix.go

```diff
@@ -24,8 +24,10 @@ func Open(name string, options ...func(*Term) error) (*Term, error) {
 		return nil, &os.PathError{"open", name, e}
 	}
 	t := Term{name: name, fd: fd}
-	if err := termios.Tcgetattr(uintptr(t.fd), &t.orig); err != nil {
+	if term, err := termios.Tcgetattr(uintptr(t.fd)); err != nil {
```

Comment:
Thanks, I original did it to avoid the need to move assignment of of the if statement.